src/main/resources/config/application-prod.yml

```diff
@@ -26,6 +26,12 @@ management:
         enabled: false
 
 spring:
+  cloud:
```

Comment:
I suggest moving these to separate `gcp` specific profile

Reply:
Thanks. Done. :+1: